test/src/getExample.js

```diff
@@ -51,14 +87,22 @@ async function upload(state, example) {
 }
 
 function updateExampleFile(state, example, recordingId) {
-  const newExampleRecordings = { ...state.exampleRecordings, [example]: recordingId };
-  fs.writeFileSync("./test/example-recordings.json", JSON.stringify(newExampleRecordings, null, 2));
+  state.exampleRecordings = { ...state.exampleRecordings, [example]: recordingId };
+  fs.writeFileSync(
+    "./test/example-recordings.json",
+    JSON.stringify(state.exampleRecordings, null, 2)
+  );
 }
 
 async function recordExample(state, example, target) {
-  const browser = target == "gecko" ? "firefox" : "chromium";
-  await record(state, browser, example);
-  const recordingId = await upload(state, example);
+  const exampleUrl = `${state.testingServer}/test/examples/${example}`;
+  const browser = target === "gecko" ? "firefox" : "chromium";
+  let recordingId;
+  await recordToFile(state, browser, exampleUrl);
+  recordingId = await upload(state, exampleUrl);
+  // await recordToCloud(state, browser, exampleUrl);
+  // recordingId = fs.readFileSync(state.exampleRecordingIdFile, "utf8").trim();
+  // fs.unlinkSync(state.exampleRecordingIdFile);
```

Comment:
`recordToCloud` isn't used.
What are the plans with this?

Reply:
No plans. This is another mode for recording examples that I experimented with. It works, but I found that in CI it's less reliable than the `recordToFile()`/`upload()` combination that we use now.